packages/manager/cypress/e2e/core/account/account-linode-managed.spec.ts

```diff
@@ -0,0 +1,181 @@
+    mockGetAccount(mockAccount).as('getAccount');
+    mockGetProfile(mockProfile).as('getProfile');
+    mockEnableLinodeManaged().as('enableLinodeManaged');
+
+    // Navigate to Account Settings page, click "Add Linode Managed" button.
+    visitUrlWithManagedDisabled('/account/settings');
+    cy.wait(['@getAccount', '@getProfile', '@getLinodes']);
+
+    ui.button
+      .findByTitle('Add Linode Managed')
+      .should('be.visible')
+      .should('be.enabled')
+      .click();
+
+    ui.dialog
+      .findByTitle('Just to confirm...')
+      .should('be.visible')
+      .within(() => {
+        cy.get('h6')
+          .invoke('text')
+          .then((text) => {
+            console.log(`h6 text: ${text.trim()}`);
+            expect(text.trim()).to.equal(
+              linodeEnabledMessageText(mockLinodes.length)
+            );
+          });
+
+        // Confirm that submit button is enabled.
+        ui.button
+          .findByTitle('Add Linode Managed')
+          .should('be.visible')
+          .should('be.enabled')
+          .click();
+      });
+
+    cy.wait('@enableLinodeManaged');
+
+    // Confirm that Cloud Manager displays a notice about Linod managed is enabled.
+    cy.findByText(linodeManagedStateMessageText, { exact: false }).should(
+      'be.visible'
+    );
+  });
+
+  /*
+   * - Confirms Cloud Manager behavior when a restricted user attempts to enable Linode Managed.
+   * - Confirms that API error response message is displayed in confirmation dialog.
+   */
+  it('restricted users cannot cancel account', () => {
+    const mockAccount = accountFactory.build();
+    const mockProfile = profileFactory.build({
+      username: 'mock-restricted-user',
+      restricted: true,
+    });
+    const errorMessage = 'Unauthorized';
+
+    mockGetAccount(mockAccount).as('getAccount');
+    mockGetProfile(mockProfile).as('getProfile');
+    mockEnableLinodeManagedError(errorMessage, 403).as('enableLinodeManaged');
+
+    // Navigate to Account Settings page, click "Add Linode Managed" button.
+    visitUrlWithManagedDisabled('/account/settings');
+    cy.wait(['@getAccount', '@getProfile']);
+
+    ui.button
+      .findByTitle('Add Linode Managed')
+      .should('be.visible')
+      .should('be.enabled')
+      .click();
+
+    ui.dialog
+      .findByTitle('Just to confirm...')
+      .should('be.visible')
+      .within(() => {
+        cy.get('h6')
+          .invoke('text')
+          .then((text) => {
+            expect(text.trim()).to.equal(linodeEnabledMessageText(0));
+          });
+
+        // Confirm that submit button is enabled.
+        ui.button
+          .findByTitle('Add Linode Managed')
+          .should('be.visible')
+          .should('be.enabled')
+          .click();
+
+        cy.wait('@enableLinodeManaged');
+
+        // Confirm that Cloud Manager displays a notice about Linod managed is enabled.
+        cy.findByText(errorMessage, { exact: false }).should('be.visible');
+      });
+  });
```

Comment:
```suggestion
    ui.dialog
      .findByTitle('Just to confirm...')
      .should('be.visible')
      .within(() => {
        cy.contains(linodeEnabledMessageText(0)).should('be.visible');

        // Confirm that submit button is enabled.
        ui.button
          .findByTitle('Add Linode Managed')
          .should('be.visible')
          .should('be.enabled')
          .click();

        cy.wait('@enableLinodeManaged');

        // Confirm that Cloud Manager displays a notice about Linode managed is enabled.
        cy.findByText(errorMessage, { exact: false }).should('be.visible');
      });
```

We can do the same thing here that we did for the other test!